index.html

```diff
@@ -496,21 +492,19 @@ <h2>
         unexpected behavior, use a scope ending in a <code>/</code>.
       </div>
       <p>
-        If the <a>application context</a>'s <a>active document</a>'s
-        [=Document/URL=] is not <a data-lt="within-scope-manifest">within
-        scope</a> of the <a>application context</a>'s manifest, the user agent
-        SHOULD show a prominent UI element indicating the [=Document/URL=] or
-        at least its <a>origin</a>, including whether it is served over a
-        secure connection. This UI SHOULD differ from any UI used when the
-        [=Document/URL=] is <a>within scope</a>, in order to make it obvious
-        that the user is navigating off scope.
+        If the [=application context=]'s [=active document=]'s [=Document/URL=]
+        is not [=within scope of a manifest=], the user agent SHOULD show a
```

Comment:
I see, this suggests the above change to remove the second _manifest_ parameter from "within scope of a manifest" was deliberate. I think that was a mistake.

This now reads as "if the document URL is within scope of **any** manifest, no need to show UI", which isn't right. This should not change; it should still read "If [the URL] is not within scope of the application context's manifest".

Aside: I previously wanted "within scope [of a manifest]" to be usable without having to write "of a manifest" every time, so it acts as an overloaded function with the same name; I can simply write "within scope of _some-url_" or "within scope of _some-manifest_" and the correct one is invoked, based on context. I thought you agreed that would be possible, by defining it as `[=URL/within scope=]` and `[=manifest/within scope=]` so that when rendered, they both just read as "within scope" but they link to the correct overload. Did this change?

Reply:
Yes, sorry, we can definitely do that. I'll put those back in.

Reply:
Ok, addressed how to use the term and made sure we are getting the processed manifest from the application context .